operator/v1/tests/insightsoperators.operator.openshift.io/AAA_ungated.yaml

```diff
@@ -14,3 +14,225 @@ tests:
         spec:
           logLevel: Normal
           operatorLogLevel: Normal
+  onUpdate:
+    - name: Should be able to update lasGatherDuration attribute with a trailing zero in the status 
+      initial: |
+        apiVersion: operator.openshift.io/v1
+        kind: InsightsOperator
+        metadata:
+          name: cluster
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+      updated: |
+        apiVersion: operator.openshift.io/v1
+        kind: InsightsOperator
+        metadata:
+         name: cluster
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+        status:
+          gatherStatus:
+            lastGatherDuration: 2m0s
+            gatherers:
+            - name: testGatherer
+              lastGatherDuration: 1m0s
+              conditions:
+              - type: Gathered
+                status: "True"
+                reason: OK
+                message: "test message"
+                lastTransitionTime: "2024-12-01T08:04:21Z"
+      expected: |
+        apiVersion: operator.openshift.io/v1
+        kind: InsightsOperator
+        metadata:
+         name: cluster
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+        status:
+          gatherStatus:
+            lastGatherDuration: 2m0s
+            gatherers:
+            - name: testGatherer
+              lastGatherDuration: 1m0s
+              conditions:
+              - type: Gathered
+                status: "True"
+                reason: OK
+                message: "test message"
+                lastTransitionTime: "2024-12-01T08:04:21Z"
+    - name: Should be able to update lasGatherDuration attribute with a leading zero in the status 
+      initial: |
+        apiVersion: operator.openshift.io/v1
+        kind: InsightsOperator
+        metadata:
+          name: cluster
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+      updated: |
+        apiVersion: operator.openshift.io/v1
+        kind: InsightsOperator
+        metadata:
+         name: cluster
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+        status:
+          gatherStatus:
+            lastGatherDuration: 0h2m
+            gatherers:
+            - name: testGatherer
+              lastGatherDuration: 0m5s
+              conditions:
+              - type: Gathered
+                status: "True"
+                reason: OK
+                message: "test message"
+                lastTransitionTime: "2024-12-01T08:04:21Z"
+      expected: |
+        apiVersion: operator.openshift.io/v1
+        kind: InsightsOperator
+        metadata:
+         name: cluster
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+        status:
+          gatherStatus:
+            lastGatherDuration: 0h2m
+            gatherers:
+            - name: testGatherer
+              lastGatherDuration: 0m5s
```

Comment:
This test almost seems to contradict a later test, can you please double check?

I think the inclusion of expected and expectedError is confusing the suite

Reply:
Ahh good catch. Thank you. Those yamls....

Reply:
Should be fixed & squashed.